test/end-to-end.js

```diff
@@ -1043,6 +1043,66 @@ describe('end-to-end', function() {
         .end(done);
     }
   });
+
+  describe('Multi-project workspace support', function() {
```

Comment:
>I am wondering what is the reason for moving this test from test/workspace to test/end-to-end?

I asked her to move it in another code review as I felt it was not exactly a unit test. Ideally, we should create a new dir (ie. test/integration) for tests that are not unit based. IMO, there should be a `test/unit/workspace.js` with two sets of tests (one set for .loadWorkspace, a seperate unit for createFromTemplate). Then the test/integration can test them working together by calling both like this test is doing. e2e probably wasn't the best place, but at the time I felt it was better than mixing it in with the basic unit tests.